### DISCUSSION TITLE
Can inline edit still be used in the latest version?

### DISCUSSION THREAD

**Discussion by otnansirk** [tinacms/tinacms#3694] [Help]
![image](https://user-images.githubusercontent.com/30102523/225026340-797f33bf-9970-4de5-afbc-a19ee97998f8.png)

Because I have a problem when use inline edit in tinacms latest version.
"tinacms": "^1.3.2"

Is the documentation still available? If so, where can I find it ?

**Comment by jamespohalloran** [comment]
What version are you upgrading from? In 2021 we started to steer away from inline editing for the time being, until we can come up with a solution that's a little less sticky than our initial approach. https://tina.io/blog/evolution-of-inline-editing/

**Reply by otnansirk** [reply]
before I use this version.

"tinacms": "^0.60.3".

**Reply by otnansirk** [reply]
There is  inline editing still available for tinacms "v1.3.2" ?

**Reply by logan-anderson** [reply]
@otnansirk While we want to revisit inline editing in the future, we are not  going to support the "old" way of inline editing.
